backend/db/migrations/20220517200503-create-song.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
         type: Sequelize.INTEGER,
         references: {
           model: 'Albums'
-        }
+        },
+        onDelete: 'SET NULL'
```

Comment:
Do you want this to set null or to delete?

Reply:
null because song can be a single without an album? that was what i was thinking

Reply:
Sounds good to me